templates/item/preview.macros.j2

```diff
@@ -31,24 +32,35 @@
   Display color palettes based on JSON map output.
 #}
 {% macro color_palette(data) %}
-<div data-sassdoc="color-palette">
-  {% for name, value in data %}
-    <div data-sassdoc-color="{{ name }}">
-      <div data-sassdoc="color-swatch">
-        <div data-sassdoc="transparency-grid"></div>
-        <div data-sassdoc="color-overlay" style="background-color: {{ value.output }};"></div>
-      </div>
-      <div data-sassdoc="color-info">
-        <strong data-sassdoc="color-name">{{ name }}</strong>
-        {% for type, color in value %}
-          <div data-sassdoc="color-value">
-            {{ color }}
+  {% call show.item_section('color-palette', 'Color Preview') %}
+    <div class="color-palette">
+      {% for value, items in data|list|groupby('value') %}
+        <div class="color-preview">
+          <div class="color-swatch">
+            <div class="transparency-grid"></div>
+            <div class="color-overlay" style="background-color: {{ value }};"></div>
           </div>
-        {% endfor %}
-      </div>
+          <div class="color-info">
+            <strong class="color-name">
+              {% for item in items %}
+                <span class="color-name-option">{{ item.key }}{% if not loop.last %},&nbsp;{% endif %}</span>
+              {% endfor %}
+            </strong>
+            {% set color = value|colors %}
+            {% if color %}
+              {% for format in ['hex', 'rgb', 'hsl'] %}
+                {% if color[format] %}
+                  <code class="color-value">{{ color[format] }}</code>
```

Comment:
More of a style note: I wonder if the inner box shadow gets to be a little much around each of these values?

hmm, just kidding. I kinda like it compared to not using the box-shadow or background:
![no-vs-yesbxshadow](https://user-images.githubusercontent.com/1581694/30134859-3b9786f2-931e-11e7-905b-d5b4aaec5365.jpg)

Reply:
I'd go either way on this. Some advantages to removing that extra padding as well… @SondraE, want to weigh in on your favorite?